modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridClock.java

```diff
@@ -21,6 +21,13 @@
  * A Hybrid Logical Clock.
  */
 public interface HybridClock {
+    /**
```

Comment:
What about checking logical part overflow? Not sure that it's worth it though.

Reply:
Overflow will add a new millisecond to physical time, so it's safe to use without checks